src/sonic-yang-models/yang-models/sonic-vlan.yang

```diff
@@ -181,10 +194,10 @@ module sonic-vlan {
 					type stypes:vlan_tagging_mode;
 				}
 			}
-            /* end of list VLAN_MEMBER_LIST */
+			/* end of list VLAN_MEMBER_LIST */
 		}
-        /* end of container VLAN_MEMBER */
+		/* end of container VLAN_MEMBER */
```

Comment:
We can remove these changes also as of now.

Reply:
Addressed